bittensor/synapses/gpt2.py

```diff
@@ -241,22 +237,24 @@ def forward(self,
         # local_hidden.shape = [batch_size, sequence_len, bittensor.__network_dim__]
         local_hidden = torch.cat([encoding, local_context], dim=2)
         local_hidden = self.hidden_layer(local_hidden)
-        output.local_hidden = local_hidden
+        
```

Comment:
So this would mean the local_hidden is not actually returned by the function call, it will mean the forward call from a remote user fails entirely